packages/back-end/src/util/environments.ts

```diff
@@ -0,0 +1,44 @@
+import cloneDeep from "lodash/cloneDeep";
+import { Environment } from "../../types/organization";
+
```

Comment:
It might make sense to put this util in the shared package, but since it's currently only used on the backend, I put it there.